### PR TITLE
Fix/hamburger

### PR DESCRIPTION
**Pull request by eouin**
Add a hamburger menu, part of story https://github.wdf.sap.corp/gd/FioriExperience/issues/128

### CODE REVIEW

File: src/components/Layout/Layout.js [hunk 1]
```diff
@@ -1,41 +1,110 @@
...
+
+class Layout extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      menuToggle: false,
+      hasScroll: false
+    };
+    this.mql = null;
```
Comment (ckoutsiaris) [inline src/components/Layout/Layout.js]: what is the mql?

File: src/components/Layout/Layout.js [hunk 2]
```diff
@@ -1,41 +1,110 @@
...
+
+  componentWillUnmount = () => {
+    window.removeEventListener('scroll', this._handleOnScroll);
+
+    this.mql && this.mql.removeListener(this.onMatchMQ);
+  };
+
+  componentDidUpdate = prevprops => {
+    if (this.props.location !== prevprops.location) {
+      if (
```
Comment (ckoutsiaris) [inline src/components/Layout/Layout.js]: maybe combine the 2 ifs?

File: src/components/LayoutHeader/Header.js [hunk 3]
```diff
@@ -1,91 +1,110 @@
...
+              <Link
+                css={{
+                  display: 'flex',
+                  marginRight: 10,
+                  height: '100%',
+                  alignItems: 'center'
+                }}
+                to='/'
+              >
+                <img src={logoSvg} alt='' width='63' height='32' />
```
Comment (ckoutsiaris) [inline src/components/LayoutHeader/Header.js]: add something in the alt property

File: src/components/Layout/Layout.js (hunk 1 above)
Comment (eouin) [reply src/components/Layout/Layout.js]: it's an object to handle the window.matchMedia() function. 'mql' for media query listener.

File: src/components/LayoutHeader/Header.js (hunk 3 above)
Comment (eouin) [reply src/components/LayoutHeader/Header.js]: done

File: src/components/Layout/Layout.js (hunk 1 above)
Comment (ckoutsiaris) [reply src/components/Layout/Layout.js]: so then maybe this.mediaQueryListener = null; :)

Comment (eouin) [reply src/components/Layout/Layout.js]: done

File: src/components/Layout/Layout.js (hunk 2 above)
Comment (eouin) [reply src/components/Layout/Layout.js]: done

**Pull request merged**